gateway/src/components/Broadcast.js

```diff
@@ -1,13 +1,95 @@
-import React from "react";
-
-export const Broadcast = (props) => {
-  // import Livepeer
-  // implement agnostic component
-  // props come from widget
-  return (
-    <textarea
-      value={JSON.stringify(props, undefined, 2)}
-      style={props.style ?? { width: "100%", height: "100%" }}
-    />
-  );
+import React, { useState } from "react";
+import { EnableVideoIcon, StopIcon } from "@livepeer/react/assets";
+import * as Broadcast from "@livepeer/react/broadcast";
+import { getIngest } from "@livepeer/react/external";
+
+export const BroadcastComponent = (props) => {
+	const [streamKey, setStreamKey] = useState(null);
+
+	const createStream = async () => {
+		try {
+			const response = await fetch("https://livepeer.studio/api/stream", {
```

Comment:
thoughts on moving this to a file w/ api's?

Reply:
After looking through rest of files, maybe it doesn't matter so much. Although, what do you think when we're planning for this next task? 

https://github.com/NEARBuilders/trylivepeer.near/issues/4

Reply:
It's definitely something that I was thinking about. Separate the components in api (for fetching), create (create a stream for Broadcast or create a playbackId for a video) and displaying (the Player and Broadcast components)